arbitrum-docs/node-running/how-tos/running-a-validator.mdx

```diff
@@ -6,41 +6,25 @@ content-type: how-to
 
 Some Arbitrum nodes will choose to act as validators. This means that they watch the progress of the rollup protocol and perhaps also participate in that protocol to advance the state of the chain securely.
 Not all nodes will choose to do this. Because the rollup protocol doesn’t decide what the chain will do but merely confirms the correct behavior that is fully determined by the inbox messages, a node can ignore the rollup protocol and simply compute for itself the correct behavior.
-Here we describe different types of validators and provide instructions on how to run them.
+Here we describe different strategies that validators follow and provide instructions on how to run them.
 
-### Types of validators
+### Validation strategies
 
-- Currently, the ability to post assertions on-chain for mainnet Arbitrum chains is whitelisted. However, anyone can run a validator in `Watchtower` mode which will immediately log an error if an on-chain assertion deviates from the locally computed chain state
-- Full list of validator strategies:
-  - `Watchtower`
-    - If local state disagrees with on-chain assertion, simply log an error (no wallet required, gas never spent on-chain)
-  - `Defensive` (Whitelist required)
+- Currently, the ability to post assertions on-chain for mainnet Arbitrum chains is allowlisted.
+- Here's a full list of validation strategies:
+  - `Defensive` (allowlist required)
     - Post stake and create challenge if local state disagrees with on-chain assertion (wallet required, will only post stake on-chain if bad assertion found)
-  - `StakeLatest` (Whitelist required)
+  - `StakeLatest` (allowlist required)
     - Stay staked on latest assertion and challenge any bad assertions found (wallet required, always staked, uses some gas every time new assertion created)
-  - `ResolveNodes` (Whitelist required)
+  - `ResolveNodes` (allowlist required)
     - Stay staked on latest assertion, resolve any unconfirmed assertions and challenge any bad assertions found (wallet required, always staked, uses some gas every time unconfirmed assertion resolved or new assertion created)
-  - `MakeNodes` (Whitelist required)
+  - `MakeNodes` (allowlist required)
     - Continuously create new assertions, challenging any bad assertions found (wallet required, always staked, most expensive node to run)
     - Note that if there is more than one `MakeNodes` validator running, they might all try to create a new assertion at same time. In that case, only one will be successful, and the others will have still spent gas on reverted calls that didn't do anything.
+  
+- There's one more validation strategy that is not allowlisted and is available for both validators and full nodes: `Watchtower`. A node in `Watchtower` mode will immediately log an error if an on-chain assertion deviates from the locally computed chain state. It doesn't require a wallet, as it never takes any action on-chain. This strategy is enabled by default in all nodes.
```

Comment:
not restricted to just "full" nodes, any type of node including full and archive nodes run in watchtower mode by default